src/modules/toast/index.ts

```diff
@@ -4,4 +4,7 @@ export enum AlertMsg {
   ERROR = 'Transaction failed',
   ERROR_DUPLICATED_TX = 'Transaction failed due to duplicate transactions on PolkaSafe; please approve or reject them before creating a new transaction',
   COMPLETED_HASH = 'Completed at transaction hash',
+  MINIMUM_BALANCE = 'You do not have enough tokens to pay the transaction fee',
 }
+
+export const REQUIRED_MINIMUM_BALANCE: number = 0.05;
```

Comment:
The number gets too big, @impelcrypto asked for it to be this way.

Reply:
@bobo-k2 
Please refer to [this](https://github.com/AstarNetwork/astar-apps/pull/1116#discussion_r1446141900)

I personally prefer to use ETH instead of WEI

Reply:
By converting to ETH you are loosing a number precision (in our case balance expressed in wei has 18 decimal places, while balance expressed in ETH/number has 15 to 17 decimal places) and some edge cases will not work. [ref](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number)

Reply:
Ok, then we should use WEI for this case. We might want to use `bigint` to compare it with the balance rather than Number. @gluneau

Reply:
Let's use `ethers.utils.formatUnits(useableBalance, 18)`

Reply:
@gluneau Or you can use `ethers.utils.formatEther(wei)` so that you don't need to pass `18`.